### ISSUE TITLE
Continue investigating accessibility issues for the ‘Search this area of the map’ button

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#23057]
## Description
During review of https://github.com/department-of-veterans-affairs/va.gov-cms/issues/21028#issuecomment-3403406509 noted a few issues with the 'Search this area of the map' button. Need to continue investigating and create tickets to fix the issues found.

<details>
<summary>
Current draft notes:
</summary>

### Current behavior
- After zooming or dragging map, a new buttons appears ‘Search this area of the map’. 
  - Did not hear the button announced when it appeared on screen. 
    - Button component has an arialive=assertive. Lack of announcement may be due to typo in attribute name.
- Additionally it looks like the live region is not in the DOM before the button appears on screen (so browser won’t recognize the change in content to the live region) 
- When you click the button, focus moves to results summary
- When you zoom too far out, but changes to a disabled ‘Zoom in to search’ button
  - The change in button content is not announced. 

### Notes
- After clicking the ‘Search this area of the map’, focus moves to results summary.
  - I did see a loss of focus where focus resets at the top of the page, but unable to reproduce…So something to keep an eye on.
- The ‘Zoom in to search’ button is disabled. 
  - Disabled buttons are discouraged. 
  - Disabled button can still be tabbed to. Should it be?
  - aria-disabled=true
  - Also change to this button is not announced. 
- It is focusable, but at the end of the map order (after ‘mapbox’ link) even though it appears at the top of the map. Focus order/visual order mismatch. 
  - Focus order does not seem stable? Is it due to the hidden map footer links getting focus when offscreen? 
  - Also seems to differ between desktop and mobile
- This button has a span in the component with a role=status 
  - When does that get populated and announced?
  - What is it announcing? 
  - I’ve never seen it populated.

### Recommendations
1. Ensure live region announces when the button appears or updates
Additions or updates to the page need to be announced via screenreader. Currently, when the button is displayed, there is no announcement. 

- This is due to a typo.
  - Fix arialive typo. Hypothesis: This is causing the button to not announce when it displays. I can’t test it in devtools because the component rehydrates and wipes my changes.
  - Change it to aria-live.
- Ensure the live region is in the DOM _before_ content is populated/updated.
- Confirm it announces when it appears on screen.

2. Chill out the announcement
Now that it’s announcing, it will interrupt other screenreader speech. aria-live=assertive is probably more aggressive than we need. It’s not an urgent enough message to necessitate that. 

- Change `aria-live=assertive` to `aria-live=polite`
</details> 

## AC / Expected behavior
- [ ] Notes documenting current state and recommendations
- [ ] Follow up tickets created
